prac_05/do_from_scratch.py

```diff
@@ -1,3 +1,5 @@
+#function to count words in a string
+
 word_count = {}
 string_to_count = str(input("Enter string"))
 string_to_count = string_to_count.strip().split(" ")
```

Comment:
The split method, by default, separates at white-spaces, so do you need to specify the white-space here?

Reply:
You would only specify the `" "` (space) value if you did *not* want to split at other whitespaces, e.g. new lines or tabs.